sql_prep/models/achievements/agg__gastarbeiter.sql

```diff
@@ -0,0 +1,73 @@
+with base as (
+    select * from {{ ref('base__debater_debata') }}
+),
+
+team_club_count as (
+    select
+        debata_id,
+        is_affirmative_speaker,
```

Comment:
It's grouping by the field `is_affirmative_speaker`, which has two values - `true` (if you're on the Affirmative team)/`false` (if you're on the Negative). It's equivalent to grouping by a field called `side` that was `affirmative`/`negative` - but since we only have the former field (for now), I used that.

Reply:
I still do not get why it is necessary information for the achievement.